src/botPage/view/blockly/blocks/tools/webhook.js

```diff
@@ -0,0 +1,59 @@
+import { translate } from '../../../../../common/i18n';
+import { expectValue } from '../shared';
+
+Blockly.Blocks.webhook = {
+    init() {
+        this.jsonInit({
+            message0: translate('Webhook URL : %1'),
+            args0   : [
+                {
+                    type: 'input_value',
+                    name: 'WEBHOOK_URL',
+                },
+            ],
+            message1: translate('Payload : %1'),
+            args1   : [
+                {
+                    type: 'input_statement',
+                    name: 'WEBHOOK_PAYLOAD',
+                },
+            ],
+            colour           : '#dedede',
+            previousStatement: null,
+            nextStatement    : null,
+            tooltip          : translate('Send payload to URL'),
+        });
+    },
+    onchange: function onchange(ev) {
+        if (!this.workspace || this.isInFlyout || this.workspace.isDragging()) {
+            return;
+        }
+
+        if (ev.type === Blockly.Events.MOVE) {
+            let currentBlock = this.getInputTargetBlock('WEBHOOK_PAYLOAD');
+
+            while (currentBlock !== null) {
+                if (currentBlock.type !== 'webhook_payload') {
+                    currentBlock.unplug();
```

Comment:
You can pass a `true` value to the `unplug` function so the stack of blocks connected to the block being unplugged (these are likely `WEBHOOK_PAYLOAD` blocks) are re-added to the statement. https://developers.google.com/blockly/reference/js/Blockly.Block#unplug

```suggestion
                    currentBlock.unplug(true);
```